src/sentry/apidocs/parameters.py

```diff
@@ -46,6 +91,51 @@ class ISSUE_ALERT_PARAMS:
     )
 
 
+class VISIBILITY_PARAMS:
+    QUERY = OpenApiParameter(
```

Comment:
Is this a visibility-only param? I think we use it in several places outside of visibility.

Reply:
This is a specific `Query` to Visibility (see the bit about the search syntax)